plugins/woocommerce/client/legacy/js/frontend/checkout.js

```diff
@@ -560,10 +560,20 @@ jQuery( function( $ ) {
 						// Detach the unload handler that prevents a reload / redirect
 						wc_checkout_form.detachUnloadEventsOnSubmit();
 
+						// This is just a technical error fallback. i18_checkout_error is expected to be always defined and localized.
+						var errorMessage = errorThrown;
+
+						if (
+							typeof wc_checkout_params === 'object' &&
```

Comment:
This will be true if `wc_checkout_params` is `null`. `typeof null` is actually still `object`. You need to add a null check there.